src/main/java/frc/robot/utilities/TOFSensorSimple.java

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.utilities;
+
+import com.playingwithfusion.TimeOfFlight;
+import edu.wpi.first.util.sendable.SendableBuilder;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.button.Trigger;
+import java.util.function.BooleanSupplier;
+
+/** Add your docs here. */
+public class TOFSensorSimple {
+  private TimeOfFlight Sensor;
+  private double threshold;
+
+  public TOFSensorSimple(int ID, double threshold) {
+    Sensor = new TimeOfFlight(ID);
+    this.threshold = threshold;
+  }
+
+  public double getDistance() {
+    return Sensor.getRange();
+  }
+
+  public BooleanSupplier isBeamBroke() {
+    return () -> (getDistance() < threshold);
+  }
+
+  public Trigger getBeamTrigger(Command action) {
+    return new Trigger(isBeamBroke());
+  }
+
+  public void identify() {
+    Sensor.identifySensor();
+  }
+
+  public void putBuilder(SendableBuilder builder) {
+    Sensor.initSendable(builder);
+  }
```

Comment:
I don't think this is what you want to do. 

initSendable() is so Smartdashboard or Shuffleboard know how to add an object. But our code shouldn't call it directly. 

Instead you can do something like:
` myShuffleboardTab.add("ToF Unique Name", this);` 
in the constructor, or add a method to facilitate setting this up if we don't want every ToF on NetworkTables.

Reply:
It might just be better for the subsystems that have the sensors to do that themselves